yarn-project/simulator/src/public/app_logic_phase_manager.ts

```diff
@@ -25,16 +25,15 @@ export class AppLogicPhaseManager extends AbstractPhaseManager {
   }
 
   override async handle(tx: Tx, previousPublicKernelOutput: PublicKernelCircuitPublicInputs) {
+    this.log.verbose(`Processing tx ${tx.getTxHash()}`);
     // add new contracts to the contracts db so that their functions may be found and called
     // TODO(#4073): This is catching only private deployments, when we add public ones, we'll
     // have to capture contracts emitted in that phase as well.
     // TODO(@spalladino): Should we allow emitting contracts in the fee preparation phase?
-    this.log.verbose(`Processing tx ${tx.getTxHash()}`);
-    // add new contracts to the contracts db so that their functions may be found and called
     // TODO(#6464): Should we allow emitting contracts in the private setup phase?
     // if so, this should only add contracts that were deployed during private app logic.
     await this.publicContractsDB.addNewContracts(tx);
-    const [kernelInputs, publicKernelOutput, newUnencryptedFunctionLogs, revertReason, returnValues, gasUsed] =
+    const { publicProvingInformation, kernelOutput, newUnencryptedLogs, revertReason, returnValues, gasUsed } =
```

Comment:
Fine with me, but what is the reasoning for rename `publicKernelOutput`->`kernelOutput`

Reply:
Explained in another comment.